File: src/main/java/com/gmail/borlandlp/minigamesdtools/util/TimeUtils.java
package com.gmail.borlandlp.minigamesdtools.util;

public class TimeUtils {
    public static String seconds2TimeF(int secs) {
        int minutes = (secs / 60);
        int seconds = secs - (minutes * 60);

        return minutes > 0 ? minutes + " мин., " + seconds + " сек." : seconds + " секунд(ы)";
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/util/TimeUtils.kt
package com.gmail.borlandlp.minigamesdtools.util

object TimeUtils {
    fun seconds2TimeF(secs: Int): String {
        val minutes = secs / 60
        val seconds = (secs - minutes) * 60

        return if (minutes > 0) "$minutes мин., $seconds сек." else "$seconds секунд(ы)"
    }
}